schunk_gripper_interfaces/srv/StrongGripAtPositionGPE.srv

```diff
@@ -0,0 +1,15 @@
+int16 force    # Percentage points of the gripper's max. gripping force.
+               # EGU 50, 60, 80: 101 ≤ gripping force [%] ≤ 200
+               # EGU 70: 101 ≤ gripping force [%] ≤ 150
+
+bool use_gpe  # Whether to activate grip force and position maintenance.
+              # Will only be used when available in the gripper.
+              # Defaults to False.
+
+bool outward  # Whether to open the gripper for grasping.
```

Comment:
@PrajwalMangaluru 
I think that's not used here. If I'm not mistaken, that's replaced by the (absolute) position argument that decides where the jaws go. What do you think?